src/EnergyPlus/LowTempRadiantSystem.cc

```diff
@@ -1637,6 +1583,52 @@ namespace LowTempRadiantSystem {
 
     }
 
+    void RadiantSystemBaseData::errorCheckZonesAndConstructions(bool &errorsFound,
+                                                                std::string const& cAlphaField3,
+                                                                std::string const& alpha3,
```

Comment:
Nit-picking, but `alpha3` and `alpha1` arguments aren't needed here. They're already available in `this->Name` and `this->ZoneName`.

`cAlphaField3` seems unnecessary as well, it's "Zone Name" and the message would read better as `"...Radiant System in Zone=ZoneABC` anyway.

Finally, maybe it would be useful to add the object type name to the base class struct and set that during input? Or set it as a const in the various specific classes? Might be a good pattern everywhere - for objects to know their class name? @Myoldmopar @mitchute ? Then this wouldn't need any arguments. Looks like [plant components](https://github.com/NREL/EnergyPlus/blob/e884660c04f1cc6f14ff32f105ee650637069b7f/src/EnergyPlus/Plant/Component.hh#L65) have a `TypeOf` variable.

Reply:
@mjwitte Ok, I've eliminated everything except the object type passed variable and the errorsFound variable.  So, this is much better.  While I agree that it would be nice to keep system type, it would take some work to get that engineered out of the radiant code and to the way you want it.  Undoubtably, it would improve some aspects of the code, and it would be a better pattern.  But is this necessary here and now or after someone looks at all of the code and establishes the "standard"?  Anyway, changes will show up in the next commit.

Reply:
Actually, let's do this.  I will simply get rid of the need for this variable here.  The error message, instead of being:

"Occurs in " + currentModuleObject + " = " + this->Name

Has been changed to:

"Occurs in Low Temperature Radiant System = " + this->Name

The other error message has also been rewritten, eliminating the need for the system type to be passed.  Since there are only three low temperature radiant system types, it seems reasonable that with the name of the system, the surface, and the zone that a user could easily identify the source of the issue.  So, this should take care of the issue without any need to do a bunch of rewriting/reworking of the code.  Will show up in the next commit.